### ISSUE TITLE
Nixpkgs needs to follow system

### ISSUE THREAD

**Issue by Noodlez1232** [lytedev/slippi-nix#10]
When building the flake, if nixpkgs is out of date in slippi-nix, it fails to initialize video due to a versioning mismatch with the AppImage's Vulkan/OpenGL and the system's. While this may not be important when doing the home-manager/NixOS configuration (you can just do `inputs.nixpkgs.follows = "nixpkgs"`), it is important when the flake is just invoked from the CLI (`nix run github:lytedev/slippi-nix#slippi-netplay`), since this may fail.

I think the only solution here is to update `flake.lock` often, perhaps a CI action plus some tests, but I'm not 100% sure here. For sure, a warning is needed in the README. I was running into this problem before when I was adding the beta netplay.

**Comment by lytedev** [comment]
Since this is a flake, isn't the best way to use `inputs.*.follows`? This is more-or-less what I have in my main flake:

```nix
{
  inputs = {
    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";

    home-manager.url = "github:nix-community/home-manager";
    home-manager.inputs.nixpkgs.follows = "nixpkgs";

    slippi.url = "github:lytedev/slippi-nix";
    slippi.inputs.nixpkgs.follows = "nixpkgs";
    slippi.inputs.home-manager.follows = "home-manager";

    # ...
  };
  # ...
}
```

**Comment by Noodlez1232** [comment]
Yes this is true for a permanent solution, but for the `nix run github:lytedev/slippi-nix#slippi-launcher` command, it doesn't quite work because nixpkgs might be out of date, or misaligned with the user's nixpkgs.

Again, I'm not sure exactly what to do about it, though...
I wonder if removing the nixpkgs input would make it follow the input from the user's flake registry (therefore making it use the user's nixpkgs version)

**Comment by lytedev** [comment]
Maybe a different package intended purely for "nix run" that uses <nixpkgs> would suffice?

**Comment by Noodlez1232** [comment]
Hmmm. I'm not quite sure. Unfortunately it depends on the system's libraries for this (the mesa versions need to align, etc.). I very honestly don't know how to solve this other than to remove the `nix run` capability (so you don't have to support it). It does suck because it'd be a super easy way to play it, but it doesn't quite work.

A single thought I had was to have the `run` package run some sort of script that symlinks the files that home-manager would've symlink'd??
